health_vara.py

```diff
@@ -4,6 +4,17 @@
 from trytond.model import fields, DeactivableMixin
 from trytond.pool import Pool, PoolMeta
 from trytond.pyson import Bool, Eval
+import re
+
+
+def normalise_mobile_number(value):
+    # treat whitespace-only values the same as None
+    value = None if (value is None) or (value == '') or (value.isspace()) else value
+
+    # replace leading '00' with '+' to standardise international numbers
+    value = re.sub('^00', '+', value) if value else None
+
+    return value
```

Comment:
part 5.
I couldn't do much for the phone number yet, we might want to revisit making it a text field with no validation.
I have added a help string to the field, and here I replace '00' with '+' to add a little more flexibility but they will still need to specify country code.